test/errors/useImportNotModule-detailed.good

```diff
@@ -0,0 +1,7 @@
+─── error in useImportNotModule.chpl:8 [UseImportNotModule] ───
+  Attempted to use 'someProc' which is not a module.
+  Target named here:
+      |
+    8 |   use M.someProc;
+      |
```

Comment:
Here, I'd add a message like "Only modules and enums [and something else, or maybe not enums?] can be used with 'use'/'import' statements".

Reply:
From [here](https://chapel-lang.org/docs/language/spec/modules.html#using-modules) and [here](https://chapel-lang.org/docs/language/spec/modules.html#importing-modules) it seems like it's just modules and enums. I'll add that